.github/ISSUE_TEMPLATE/4-chore.md

```diff
@@ -0,0 +1,25 @@
+---
+name: Chore
+about: Work on a chore needing completion
+labels: 'type: chore'
+---
+
+<!--
+Please use this template to create your chore issue. Sometimes you see and out-of-date README, sometimes the spelling of a word is off, sometimes you spot deeply nested 7-layer for-loop could be better handled another way; use this template for your non-bug-related fixes/updates/refactors.
```

Comment:
Would be good to reduce the use of the word "sometimes" here (also using and rather than an). i.e `Sometimes you see an out-of-date README, the spelling in a word is off, or you spot a...`

Also should be `non-bug related`

Reply:
Fixed.